src/main/java/org/dcsa/ebl/service/impl/ShippingInstructionServiceImpl.java

```diff
@@ -291,6 +298,18 @@
             "number of originals is required for non electronic shipping instructions.");
       }
     }
+    if (Objects.nonNull(shippingInstructionTO.getIsElectronic())
```

Comment:
Please add a comment for this validation as is done for all other validation rules

Reply:
Done